### ISSUE TITLE
[MNT] `label_smoothing` parameter of `CrossEntropyLoss` should not be exposed in soft labelling loss functions

### ISSUE THREAD

**Issue by victormvy** [ayrna/dlordinal#58]
### Describe the issue

`label_smoothing` parameter of `CrossEntropyLoss` applies label smoothing by mixing the one-hot targets with a uniform distribution. However, in soft labelling loss functions, it makes no sense to mix an already soft label encoding with a uniform distribution.

### Suggest a potential alternative/fix

The `label_smoothing` parameter should be removed from

- `PoissonCrossEntropyLoss`
- `BinomialCrossEntropyLoss`
- `ExponentialCrossEntropyLoss`
- `BetaCrossEntropyLoss`
- `TriangularCrossEntropyLoss`
- `GeneralTriangularCrossEntropyLoss`

Then, the value passed to the `CrossEntropyLoss` when initialising the `ce_loss` attribute should be `0`.

### Additional context

Thank you!

**Comment by victormvy** [comment]
`label_smoothing` parameter has been removed from the proposed classes' init methods.
